client/app/pool/pool.html

```diff
@@ -1,12 +1,15 @@
 <div class="poolFrame" ng-model="vm" ng-cloak>
   <div id="candidateName">{{ vm.candidateName }}</div>
   <div id="candidatePic">
-    <img class="responsiveImg" ng-if="vm.isCandidateAvailable" ng-src="{{ vm.candidatePicUrl }}">
+    <img class="responsiveImg" ng-if="vm.isCandidateAvailable"
+    ng-src="{{ vm.candidatePicUrl }}" imageonload>
     <div id="noUsersMessage" ng-if="!vm.isCandidateAvailable">{{ vm.noUsersMessage }}</div>
   </div>
   <div id="controls" ng-if="vm.isCandidateAvailable">
-    <div class="control" ng-click="vm.refuseCandidate()">NO</div>
-    <div class="control" ng-click="vm.approveCandidate()">YES</div>
+    <div class="control" ng-class="{ 'inactive': !candidateImageLoaded }"
+    ng-click="!candidateImageLoaded || vm.refuseCandidate()">NO</div>
+    <div class="control" ng-class="{ 'inactive': !candidateImageLoaded }"
+    ng-click="!candidateImageLoaded || vm.approveCandidate()">YES</div>
```

Comment:
why do you need ng-click="!candidateImageLoaded || ? can it just be  vm.approveCandidate() ? looks weird